var/spack/repos/builtin/packages/openblas/openblas-0.3.10-appleclang12.patch

```diff
@@ -0,0 +1,384 @@
+From 1a05523709567d54273fe1000ae4319df437737c Mon Sep 17 00:00:00 2001
```

Comment:
Please generate the patch file from this commit: https://github.com/xianyi/OpenBLAS/commit/f42e84d46c52f4ee1e05af8f365cd85de8a77b95

The one you used appears to have been a temporary one which does not belong to the `develop` branch of OpenBLAS.

Reply:
You can also add the patch via URL and sha256.

Reply:
I only tested that the patch applies and did not do another test build, I assume it still works fine, but can do another test build tomorrow at the earliest. Do you want me to or merge as is?

Reply:
> You can also add the patch via URL and sha256.

Just for my understanding: Will a URL-based patch be included in the source cache?

Reply:
Yes, a URL-based patch would be cached just like resources.